packages/dd-trace/src/encode/coverage-ci-visibility.js

```diff
@@ -58,24 +76,16 @@ class CoverageCIVisibilityEncoder extends AgentEncoder {
   }
 
   makePayload () {
-    const form = new FormData()
-
-    let coverageFileIndex = 1
-
-    for (const coverageBuffer of this.codeCoverageBuffers.slice(0, MAXIMUM_NUM_COVERAGE_FILES)) {
-      const coverageFilename = `coverage${coverageFileIndex++}`
-      form.append(
-        coverageFilename,
-        coverageBuffer,
-        {
-          filename: `${coverageFilename}.msgpack`,
-          contentType: 'application/msgpack'
-        }
-      )
-    }
-    // 'event' is a backend requirement
-    form.append('event', JSON.stringify({}), { filename: 'event.json', contentType: 'application/json' })
-    this.codeCoverageBuffers = this.codeCoverageBuffers.slice(MAXIMUM_NUM_COVERAGE_FILES)
+    this.form.append(
+      'event',
+      JSON.stringify({ dummy: true }),
```

Comment:
Was this intended for a test?

Reply:
no, sadly this is an intake limitation. It's not enough for this to be an empty dictionary, it has to have a key

Reply:
Can you add a comment to make that clear? Otherwise it really looks unintended.